### ISSUE TITLE
did you try to start with maximum possibile cache size

### ISSUE THREAD

**Issue by rouniuyizu** [facebookresearch/adaptive-span#8]
Just curious: would the results (the final span length and pred accuracy) still hold if you start with max cache size -- initialized by torch.ones (need to reduce S) or random number between 0~1 instead of zeros?

https://github.com/facebookresearch/adaptive-span/blob/a8d90b8a8481ef1ae50a73b696c290aa88d34744/adaptive_span.py#L34

**Comment by tesatory** [comment]
No we haven't tried that, but we did tried starting with 50% of max span, and the results were similar.

**Comment by rouniuyizu** [comment]
thank you
